src/Controller/Controller.php

```diff
@@ -13,7 +13,7 @@
 namespace App\Controller;
 
 use Doctrine\Persistence\ManagerRegistry;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller as BaseController;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController as BaseController;
```

Comment:
We can remove the alias now, as it doesn't conflict with the `Controller` class below anymore.